src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/VTableSliceNode.cs

```diff
@@ -19,6 +19,7 @@ internal class VTableSliceNode : DependencyNodeCore<NodeFactory>
         TypeDesc _type;
         bool _shouldBuildFullVTable;
         List<MethodDesc> _slots = new List<MethodDesc>();
+        bool _slotsCommitted;
```

Comment:
You could put this (and the uses) under an `#if DEBUG` so that it doesn't waste space in the object in Release builds.